developers/weaviate/configuration/indexes.md

```diff
@@ -102,20 +102,23 @@ Note that the vector index type only specifies how the vectors of data objects a
 
 ## Inverted index
 
-### Configure the inverted index
+### Configure the inverted index - for `text`
 
-In contrast to the ANN index, there are very few configurable parameters for an inverted index. It's on or it's off and indexes on a property level.
+There are two indexes for filtering or searching the data, where the first (filterable) is for building a fast, Roaring Bitmaps index, and the second (searchable) index is for a BM25 or hybrid search.
```

Comment:
filterable index allows only filtering, searchable index allows searching and filtering (though not as fast as RBM filterable). So if user does not care about filter performance, would like to save some disk space by having just 1 instead of 2 indexes, or want faster import (updating 1 index is faster than 2), he can have just searchable.

Reply:
Thanks! That's really useful info.